### PR TITLE
SB-238 Implement triggering of index rebuilding via REST

### PR DESCRIPTION
**Pull request by kate-novik**

### CODE REVIEW

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/ArtifactIndexesController.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        if (getStorage(storageId) == null)
+        {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND)
+                                 .body("The specified storageId does not exist!");
+        }
+        try
+        {
+            artifactIndexesService.rebuildIndexes(storageId);
+
+            return ResponseEntity.ok("The Indexes was successfully rebuilt!");
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/ArtifactIndexesController.java]: @kate-novik ,

Could you please correct the response string to:

    return ResponseEntity.ok("The indexes were successfully re-built!");

Here and across the other places below...?

Thanks! :)

**Comment by carlspring** [comment]
@kate-novik ,

Would you mind removing the unused imports reported by Sonar? You can ignore the ones about the cyclomatic complexity.

**Comment by kate-novik** [comment]
Yes, of course.

7 дек. 2016 г. 2:01 ПП пользователь "Martin Todorov" <
notifications@github.com> написал:

**Comment by carlspring** [comment]
@cs-devops-bot rebuild.

**Comment by carlspring** [comment]
Thanks for your contribution! :)

**Pull request merged**